### PR TITLE
add worldwide poi datasets

### PR DESCRIPTION
**Pull request by nlehuby**
* a national museum dataset from wikidata
* an airport dataset from openflights

### CODE REVIEW

**Comment by missinglink** [comment]
hi @nlehuby the 2nd line in the museums file seems to be a duplicate of a line from the airports file:

> Goroka Airport;4;-6.081689834590001,145.391998291,500;

**Comment by nlehuby** [comment]
Thank you @missinglink. I've amended the commit to remove the duplicate.

File: geocoder_tester/world/poi/test_airports.csv [hunk 1]
```diff
@@ -0,0 +1,7185 @@
...
+Fassberg Air Base;4;Germany;52.919399,10.185827,1500;OurAirports;\N
+Grafenwohr Army Air Field;4;Germany;49.698699951171875,11.940199851989746,1500;OurAirports;\N
+Hanau Army Air Field;4;Germany;50.169201,8.96159,1500;OurAirports;ZNF
+Hohenfels Army Air Field;4;Germany;49.218101501464844,11.836099624633789,1500;OurAirports;\N
+Kitzingen Army Air Field;4;Germany;49.743099212646484,10.200599670410156,1500;OurAirports;\N
+Nordholz Naval Airbase;4;Germany;53.7677001953,8.658499717709999,1500;OurAirports;\N
+Diepholz Air Base;4;Germany;52.585556,8.342222,1500;OurAirports;\N
+Geilenkirchen Air Base;4;Germany;50.9608,6.04242,1500;OurAirports;GKE
+Hohn Airport;4;Germany;54.312198638916016,9.538169860839844,1500;OurAirports;\N
+Jever Air Base;4;Germany;53.533501,7.88867,1500;OurAirpor
```
Comment (yohanboniface) [inline geocoder_tester/world/poi/test_airports.csv]: I think a bit weird to have "Airport" (in English) in every name. Isn't a local name available?

Comment (nlehuby) [reply geocoder_tester/world/poi/test_airports.csv]: I don't speak the Finnish language; if I'm looking for the Enontekiö Airport, I would definitly search for "Enontekiö Airport" and not for "Enontekiön lentoasema" (which is the local name).

This test file is about internationalization handling.

Comment (yohanboniface) [reply geocoder_tester/world/poi/test_airports.csv]: I wonder if we should tag it in some way then, given it's sort of "typed" ("rue de Rivoli" vs "Rivoli Street").
Or maybe adding `lang=en`.
Not a big deal though, and given we are under `world` we may merge as is and see later if it's an issue for some workflow.

Comment (nlehuby) [reply geocoder_tester/world/poi/test_airports.csv]: you're right. I've added the lang column.

**Pull request merged**